chat-plugins/helptickets.js

```diff
@@ -446,7 +446,17 @@ exports.pages = {
 exports.commands = {
 	requesthelp: 'helpticket',
 	helprequest: 'helpticket',
-	report: 'helpticket',
+	report: function (target, room, user, connection) {
+		let message = toId(target);
+		if (message.includes('list') ||
```

Comment:
I don’t see why these need to be on separate lines either

Reply:
you're right i started it that way and didn't realise there were so few commands lol